src/index.js

```diff
@@ -3,11 +3,14 @@ import ReactDOM from 'react-dom';
 import App from './components/app/App';
 import { GlobalStyle } from './components/global/styledGlobal';
 import reportWebVitals from './reportWebVitals';
+import { HashRouter } from 'react-router-dom';
 
 ReactDOM.render(
   <React.StrictMode>
-    <GlobalStyle />
-    <App />
+    <HashRouter>
```

Comment:
I can't remember, because it was awhile ago, but I used browserRouter for a project and had issues that disappeared when I did a deep dive online and discovered hashrouter.

Not a great answer, I know.

Reply:
I guess we should figure out which one to use and make a good decision here. The nav links are just going to navigate us down to the necessary sections so maybe we don't even need a router? I'm not 100% sure.